test/test-pages/buzzfeed-1/expected.html

```diff
@@ -1,9 +1,5 @@
 <div id="readability-page-1" class="page">
     <div id="buzz_sub_buzz">
-        <div id="superlist_3758406_5547137" rel:buzz_num="1">
-            <h2>The mother of a woman who took suspected diet pills bought online has described how her daughter was “literally burning up from within” moments before her death.</h2>
-            <p> <span>West Merica Police</span></p>
-        </div>
```

Comment:
Hm, shouldn't this be staying here?

Reply:
yes, that will stay in moment when I keep first headers.